### DISCUSSION TITLE
posts/tutorial-configuracion-openmediavault/

### DISCUSSION THREAD

**Discussion by giscus** [marcosruiz/marcosruiz.github.io#16] [General]
# posts/tutorial-configuracion-openmediavault/

Cambiar contraseña del usuario admin

https://marcosruiz.github.io/posts/tutorial-configuracion-openmediavault/

**Comment by Romvict19** [comment]
Hola Marco, 
Muchas gracias por este documento de guía sinceramente me has salvado. Pero podrías poner la configuración de los usuarios y grupos. Es que me genera dudas este apartado, sobre todo el remoto.
